components/select_team/components/select_team_item.jsx

```diff
@@ -85,13 +85,14 @@ export default class SelectTeamItem extends React.PureComponent {
                     onClick={canJoin ? this.handleTeamClick : null}
                     className={canJoin ? '' : 'disabled'}
                 >
+                    <span className='signup-team-dir__name'>{team.display_name}</span>
+                    {canJoin}
```

Comment:
This shouldn't be here, I think.

Reply:
Where should it be?

Reply:
I mean, the `canJoin` at the bottom with the `{canJoin && icon}` was to show only the arrow when you can join the team. Now the arrow is always shown, and here is show a `true` string. Is not a huge deal to remove the `{canJoin && icon}` and show always the icon, because for now we only list joinable teams, but in the future we probably want to show not-joinable teams.